### PR TITLE
Output ES2020

### PR DESCRIPTION
**Pull request by justinfagnani**
- Regen package-lock to pick up rollup-plugin-filesize 9.1.0
- Update most tsconfigs to target es2020.
- Update Firefox ESR in tests to latest version

### CODE REVIEW

File: packages/lit-html/static.d.ts [hunk 1]
```diff
@@ -0,0 +1,47 @@
+/**
```
Comment (aomarks) [inline packages/lit-html/static.d.ts:1]: Shouldn't be checked in?

**Comment by aomarks** [comment]
> * Update most tsconfigs to target es2020. `localize` is left at es2018

Why not update localize too?

File: packages/lit-html/static.d.ts (hunk 1 above)
Comment (justinfagnani) [reply packages/lit-html/static.d.ts:1]: er... I need to fix the .gitignore

**Comment by justinfagnani** [comment]
> Why not update localize too?

localize was the only package targeting es2018, I thought maybe for a reason. Updating the language level is a breaking change, is that ok? If so, I'll update.

**Comment by aomarks** [comment]
> localize was the only package targeting es2018, I thought maybe for a reason. Updating the language level is a breaking change, is that ok? If so, I'll update.

Yep it's fine

**Comment by justinfagnani** [comment]
PTAL

**Review by aomarks — APPROVED** [review]

**Pull request merged**